### DISCUSSION TITLE
✗ Error uploading to Backblaze B2  InvalidAccessKeyId: Malformed Access Key Id at throwDefaultError

### DISCUSSION THREAD

**Discussion by pguardiario** [muxinc/next-video#218] [Q&A]
I can't figure out how to set .env for backblaze, is access key the same thing as application key?

In backblaze I see:
keyID: xxxxxx
keyName: xxxxxx
applicationKey: xxxxxxxxxxxxxx

and in my .env my best guess is this which gives an error:
BACKBLAZE_ACCESS_KEY_ID=(keyID goes here?)
BACKBLAZE_SECRET_ACCESS_KEY=(applicationKey goes here?)

**Comment by luwes** [comment]
yes

keyId is BACKBLAZE_ACCESS_KEY_ID
applicationKey is BACKBLAZE_SECRET_ACCESS_KEY

try generating a new application key, not the default provided master app key

**Comment by pguardiario** [comment]
Ok thanks for the info.
I did generate a new key and I'm still getting the error but I was just trying it out, I'm not in a hurry to solve this.
